### PR TITLE
Improve XDR Disconnected Endpoints job playbook

### PR DESCRIPTION
**Pull request by ali-sawyer**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
n/a

## Description
Make the following improvements to the XDR Disconnected Endpoints playbook:
1. Add script that pages over all endpoints rather than cutting off at 30
2. Convert timestamps to human-readable timestamps for the report
3. Fix a typo
4. Gracefully handle the situation where no lost/disconnected endpoints are found

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by rshunim** [comment]
Hi @ali-sawyer,
Thanks a lot for your contribution!

Please at first fix the failures and make the build green,
you can see which validations are failed, as I see you have structure errors and you can fix them with the command 'demisto-sdk format -i <the path of the problematic file>',
add a Release Note file and upgrade the version in the pack_metadata file,
when you fix the all errors, we can proceed with reviewing your PR

Thanks a lot again

Feel free to contact me via Slack, Roei Shunim

File: Packs/CortexXDR/ReleaseNotes/6_1_71.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+
+#### Playbooks
+
+##### New: Cortex XDR disconnected endpoints
```
Comment (tomer-pan) [inline Packs/CortexXDR/ReleaseNotes/6_1_71.md:4]: Hi @ali-sawyer !
The playbook is not new. Please only add the changes made to the playbook.

**Comment by tomer-pan** [comment]
Hi @ali-sawyer 
I noticed that you added a script that "fixes" some of the "xdr-get-endpoints" command issues.
- It's not best practice to use a script to bypass command limitations and since this "XSOAR Supported" pack, we can't do this here.
- It is possible to request an enhancement (or do it as part of a contribution) that the command will do the needed things, like retrieve all endpoints, return HR timestamp, or handle an error gracefully. 

Please let me know how you want to continue, we can also discuss about it personally if you like.

**Comment by rshunim** [comment]
Closing this PR,
@ali-sawyer thinks it will be better for her to add these improvements (as @tomer-pan suggested her) to the XDR integration vs adding a custom script. She will open a new one when she is ready

**Pull request closed without merging**